lib/koans/10_processes.ex

```diff
@@ -2,17 +2,17 @@ defmodule Processes do
   use Koans
 
   koan "Tests run in a process" do
-    assert Process.alive?(self()) == ___
+    assert Process.alive?(self) == ___
```

Comment:
I think the jury is still out... I tend to use parens as well but not all the time. For `self` I probably wouldn't use them but I have no good reason for why :)